PopMovies/PopMovies/Sources/Views/MovieDetails/Views/MovieDetailsViewController.swift

```diff
@@ -26,4 +26,20 @@ class MovieDetailsViewController: PMViewController {
         view = detailsView
     }
 
+    override func viewDidLoad() {
+        getSimilarMovies()
+    }
```

Comment:
lembrar de sempre chamar o `super.viewDidLoad`
```suggestion
    override func viewDidLoad() {
        super.viewDidLoad()
        getSimilarMovies()
    }
```